addons/analytic/tests/test_analytic_account.py

```diff
@@ -70,6 +70,9 @@ def setUpClass(cls):
         })
 
     def test_aggregates(self):
+        if group_readonly := self.env.ref('account.group_account_readonly', raise_if_not_found=False):
+            # debit and credit are hidden by the group when account is installed
+            self.env.user.groups_id |= group_readonly
```

Comment:
So... in case you don't have the group, why running a test you know will fail?
Why not:
```suggestion
    if not self.env.user.user_has_group('account.group_account_readonly'):
        self.skipTest(...)
```
Also, you could just remove `debit` & `credit` below in case you don't have the group. It's probably the less ugly way to fix your problem

Reply:
Indeed prefer the second solution